micro/server.py

```diff
@@ -965,6 +966,36 @@ def post(self) -> None:
         self.set_status(HTTPStatus.CREATED) # type: ignore
         self.write(referral.json(restricted=True, include=True))
 
+    def get(self) -> None:
```

Comment:
This overwrites the original `GET` behavior (`_ReferralsEndpoint` extends `CollectionEndpoint` which returns a plain list of `Referral` objects on `get()`), which we want to keep at least for backwards compatibility. :sweat_smile: To fix it, you would move the method to its own `class ReferralSummaryEndpoint(Endpoint)` and list it in `Server.handlers` with the URL `/api/(?:analytics|stats)/referrals/summary$`.